### PR TITLE
41 more powerups

### LINKED ISSUE #41: More types of powerups
Obstacles can be destroyed with the shooting powerup. This also raises the score.

### PR DESCRIPTION
**Pull request by mpsijm**
Will close #41 

I've added two powerups: Slowness and Destroy.
- Slowness makes the Track twice as slow and slows down the soundtrack by a factor ³√2.
- Destroy makes the Logs destroyable and grants you 25 points per Log destroyed.

Have fun playing and reviewing! :D

<review, feature

### CODE REVIEW

File: src/main/java/nl/tudelft/ti2206/group9/level/CollisionHandler.java [hunk 1]
```diff
@@ -38,28 +43,40 @@ public void collide(final AbstractEntity collider,
...
+                        notifyCollide(AbstractObstacle.class);
+                    }
+                });
+        collisionMap.onCollision(Player.class, Log.class,
+                (collider, collidee) -> {
+                    if (AbstractPowerup.isActive(PowerupDestroy.class)) {
+                        collidee.selfDestruct();
+                        notifyCollide(Log.class);
+                        State.addScore(Log.VALUE);
+                        return;
```
Comment (thervh70) [inline src/main/java/nl/tudelft/ti2206/group9/level/CollisionHandler.java]: Why is this return statement present here?

< review

Comment (thervh70) [inline src/main/java/nl/tudelft/ti2206/group9/level/CollisionHandler.java]: Oh wait i see. Can't you just replace the if statement following this method with an else if statement?

**Comment by thervh70** [comment]
The changes are looking very good! I really like the new powerups!
Please merge.

< review

**Comment by mdingjan** [comment]
I've reviewed your changes and fixed one tiny Checkstyle warning.
The code is looking great and the powerups are really cool. :D 

Will merge!

<review,feature

**Pull request merged**